libraries/AP_OpticalFlow/OpticalFlow.cpp

```diff
@@ -59,7 +59,8 @@ const AP_Param::GroupInfo OpticalFlow::var_info[] = {
 
 // default constructor
 OpticalFlow::OpticalFlow(AP_AHRS_NavEKF &ahrs)
-    : _last_update_ms(0)
+    : _last_update_ms(0),
+      _ahrs(ahrs)
```

Comment:
Also, please see the reasoning above to why "comma in the same position as colon" is a good coding style.  I think the first place I saw that in use was in webkit (https://webkit.org/code-style-guidelines/#other-punctuation) back when I was doing webengines. Of course that could just trigger some bike-shedding.